docs/src/main/asciidoc/hibernate-orm-panache.adoc

```diff
@@ -1,326 +1,752 @@
 
-== First: an example
+As a Java developer, you can use Hibernate ORM with Panache to simplify persistence with your {ProductName} applications.
 
-What we're doing in Panache is allow you to write your Hibernate ORM entities like this:
+.Prerequisites
+You have a {ProductName} Maven project configured with a Java database connectivy (JDBC) datasource and Hibernate ORM.
 
-[source,java]
-----
-@Entity
-public class Person extends PanacheEntity {
-    public String name;
-    public LocalDate birth;
-    public Status status;
+ifdef::ENT[]
+* For information about creating a {ProductName} Maven project, see https://access.redhat.com/documentation/en-us///red_hat_build_of_quarkus/{ProductVersion}/html-single/creating_quarkus_applications_with_apache_maven/[_Creating Quarkus applications with Apache Maven_].
+endif::ENT[]
 
-    public static Person findByName(String name){
-        return find("name", name).firstResult();
-    }
+ifdef::COMM[]
+* For information about creating a {ProductName} Maven project, see https://quarkus.io/guides/maven-tooling[_Quarkus - Building Applications with Maven_].
+endif::COMM[]
 
-    public static List<Person> findAlive(){
-        return list("status", Status.Alive);
-    }
+* For information about configuring your {ProductName} Maven project with a JDBC datasource, see https://quarkus.io/guides/datasource[_Quarkus - Configuring your datasources_].
+* For information about configuring your {ProductName} Maven project Hibernate ORM, see https://quarkus.io/guides/hibernate-orm[_Quarkus - Quarkus - Hibernate ORM and JPA_].
 
-    public static void deleteStefs(){
-        delete("name", "Stef");
-    }
-}
-----
+For an example of a completed {ProductName} Maven project that uses the Panache extension, see the https://github.com/quarkusio/quarkus-quickstarts/tree/master/hibernate-orm-panache-quickstart[Hibernate ORM Panache Quickstart].
 
-You have noticed how much more compact and readable the code is?
-Does this look interesting? Read on!
+== Hibernate ORM
 
-NOTE: the `list()` method might be surprising at first. It takes fragments of HQL (JP-QL) queries and contextualizes the rest. That makes for very concise but yet readable code.
+//adapted from https://docs.jboss.org/hibernate/orm/6.0/userguide/html_single/Hibernate_User_Guide.html. Module will be reused in Hibernate ORM story.
 
-== Setting up and configuring Hibernate ORM with Panache
+Coupling Java object-oriented software projects with relational databases for data persistence can be cumbersome and time-consuming. Development costs are significantly higher due to a paradigm mismatch between how data is represented in objects versus relational databases. Hibernate is an Object/Relational Mapping (ORM) solution for Java environments that facilitates mapping data to and from an object model representation to a relational data model representation. It is an implementation of the Java Persistence API (JPA).
```

Comment:
Do we really have to explain Hibernate ORM in this guide? We have the Hibernate ORM guide itself that we can point to, for those who don't know it, no?

Reply:
I'll leave this to Nicole and Vidya to decide.

Reply:
Do we have that guide on the customer portal?  I don't think we do at this time so I say we should have this here.  Also, repetition isn't always a bad thing if it stops folks from having to go to multiple places for the information they need.

Reply:
I think that guide will also have to be there because the Hibernate ORM extension is productised.

Reply:
Removed and preserved sin a separate hibernate-con.adoc file o the section can be used in the Hibernate story.